docs/INSTALL.rst

```diff
@@ -70,3 +70,16 @@ and should work with teuthology. But if you want to bump up the versions of them
 please use the following command to update this file ::
 
   pip-compile -qo- | sed '/^-e / d' > requirements.txt
+
+Please upgrade pip-tool using following command ::
+
+  pip install pip-tools --upgrade
+
+if the command above fails like::
```

Comment:
@tchaikov I think this commit should not has merged, it would be enough to 'pip-compile -P pip-tools', btw, pip-compile from py2.7 and py3.x produce different output.

Reply:
is it a chicken and egg problem? also, i am not sure if we are python3 ready? so i assume it's all python2.

Reply:
@tchaikov we will never be python3 if we will not python3 have ready requirements.txt. 
Btw, https://jenkins.ceph.com/job/teuthology-pull-requests/707/console:
```
___________________________________ summary ____________________________________
  docs: commands succeeded
  py3: commands succeeded
  py27: commands succeeded
  py27-integration: commands succeeded
  flake8: commands succeeded
  openstack: commands succeeded
  congratulations :)
```

Reply:
thanks for the information. but i don't think python3 compatibility is in the scope of this PR. its purpose was to address https://tracker.ceph.com/issues/43192

Reply:
yes... but if under "it's all python2" means "it's all python2 ONLY" is getting wrong, yes we need to keep it backward compatible with py2, but we should prefer py3 compatibility over py2 only code.